flash/image/classification/data.py

```diff
@@ -121,14 +121,12 @@ def _to_numpy(img: Union[torch.Tensor, Image.Image]) -> np.ndarray:
             raise TypeError(f"Unknown image type. Got: {type(img)}.")
         return out
 
+    @_requires_extras("image")
     def _show_images_and_labels(self, data: List[Any], num_samples: int, title: str):
```

Comment:
do you need kornia here?

Reply:
basically, I aim at the granularity that the user may have installed some packages without Flash defined extras so for example for this function he does need all extras for image, right?

Reply:
The problem with granularity is that you get this sort of flow:
- try to do something e.g. create `ImageClassifier`
- get error "install torchvision"
- now create `ImageClassifier` and try to train it
- get error "install kornia"
- now create and train and try to visualize
- get error "install matplotlib"

So with this PR, instead you should just get told to install the package with "image" extras at the beginning.

Reply:
I see so maybe we can have another wrapper just for one or more particular packages?